### PR TITLE
release-notes: structure: update 6.4.0 release notes

### PR DESCRIPTION
**Pull request by xtreme-sameer-vohra**
## Contributor Checklist
- [ ] Followed [Code of conduct], [Contributing Guide] & avoided [Anti-patterns]
- [ ] [Signed] all commits
- [ ] Updated [Documentation]
- [ ] Updated [Release notes]

[Code of Conduct]: https://github.com/concourse/concourse/blob/master/CODE_OF_CONDUCT.md
[Contributing Guide]: https://github.com/concourse/concourse/blob/master/CONTRIBUTING.md
[Anti-patterns]: https://github.com/concourse/concourse/wiki/Anti-Patterns
[Signed]: https://help.github.com/en/github/authenticating-to-github/signing-commits
[Documentation]: https://github.com/concourse/docs
[Release notes]: https://github.com/concourse/concourse/tree/master/release-notes

## Reviewer Checklist

- [ ] Code reviewed
- [ ] Tests reviewed
- [ ] Documentation reviewed
- [ ] Release notes reviewed
- [ ] PR acceptance performed
- [ ] New config flags added? Ensure that they are added to the
  [BOSH](https://github.com/concourse/concourse-bosh-release) and
  [Helm](https://github.com/concourse/helm) packaging; otherwise, ignored for
  the [integration
  tests](https://github.com/concourse/ci/tree/master/tasks/scripts/check-distribution-env)
  (for example, if they are Garden configs that are not displayed in the
  `--help` text).

### CODE REVIEW

File: release-notes/v6.4.0.md [hunk 1]
```diff
@@ -40,3 +40,23 @@
...
 
 * Add `--team` flag for `fly set-pipelines` command #5805
+
+#### <sub><sup><a name="5855" href="#5855">:link:</a></sup></sub> fix
+
+* Fix tooltip on pipeline names. PR: #5855
+
+#### <sub><sup><a name="5837" href="#5837">:link:</a></sup></sub> fix
+
+* Fix fly login manual input on windows. PR: #5837
```
Comment (aoldershaw) [inline release-notes/v6.4.0.md:50]: This should be no impact - it was broken as a result of #5770

File: release-notes/v6.4.0.md [hunk 2]
```diff
@@ -40,3 +40,23 @@
...
+
+#### <sub><sup><a name="5504" href="#5504">:link:</a></sup></sub> feature
+
+* Refactor existing step structure to simplify introducing new steps. The primary user facing change is step validation messages are different. PR: #5504
+
+#### <sub><sup><a name="5729" href="#5729">:link:</a></sup></sub> feature
+
+* @tlwr updated the `set_pipeline` step to be used across teams. The [RFC #31](https://github.com/concourse/rfcs/pull/31) for this step is still technically in progress so the step is 'experimental' for now. PR: #5729
```
[the remark this answers is not in the document]
Comment (xtreme-sameer-vohra) [reply release-notes/v6.4.0.md]: done

File: release-notes/v6.4.0.md [hunk 3]
```diff
@@ -35,8 +35,28 @@
 
 #### <sub><sup><a name="5770" href="#5770">:link:</a></sup></sub> fix
 
-* `fly login` now accepts arbitrarily long tokens when pasting the token manually into the console. Previously, the limit was OS dependent (with OSX having a relatively small maximum length of 1024 characters). This has been a long-standing issue, but it became most noticable after 6.1.0 which significantly increased the size of tokens. Note that pasted token is now hidden in the console output. #5770
+* `fly login` now accepts arbitrarily long tokens when pasting the token manually into the console. Previously, the limit was OS dependent (with OSX having a relatively small maximum length of 1024 characters). This has been a long-standing issue, but it became most noticeable after 6.1.0 which significantly increased the size of tokens. Note that pasted token is now hidden in the console output. #5770
```
Comment (jomsie) [inline release-notes/v6.4.0.md:38]:
```diff
-Note that pasted token is now hidden in the console output.
+Note that the pasted token is now hidden in the console output.
```

File: release-notes/v6.4.0.md [hunk 4]
```diff
@@ -35,8 +35,28 @@
...
+
+* Refactor existing step structure to simplify introducing new steps. The primary user facing change is step validation messages are different. PR: #5504
+
+#### <sub><sup><a name="5729" href="#5729">:link:</a></sup></sub> feature
+
+* @tlwr updated the `set_pipeline` step to be used across teams. This is 'experimental' for now, please share your feedback in the [thread](https://github.com/concourse/concourse/discussions/5731). PR: #5729
+
+#### <sub><sup><a name="5034" href="#5034">:link:</a></sup></sub> feature
+
+* @evanchaoli added [OPA](https://www.openpolicyagent.org/) integration to Concourse to enable policy enforcement. The [RFC #41](https://github.com/concourse/rfcs/pull/41) for the integration is still in progress so is considered 'experimental'. PR: #5034
```
Comment (jomsie) [inline release-notes/v6.4.0.md:58]:
```diff
-the integration is still in progress so is considered 'experimental'
+the integration is still in progress so it is considered 'experimental'
```

File: release-notes/v6.4.0.md [hunk 5]
```diff
@@ -35,8 +35,28 @@
...
 
 * Add `--team` flag for `fly set-pipelines` command #5805
+
+#### <sub><sup><a name="5855" href="#5855">:link:</a></sup></sub> fix
+
+* Fix tooltip on pipeline names. PR: #5855
+
+#### <sub><sup><a name="5504" href="#5504">:link:</a></sup></sub> feature
+
+* Refactor existing step structure to simplify introducing new steps. The primary user facing change is step validation messages are different. PR: #5504
```
Comment (jomsie) [inline release-notes/v6.4.0.md:50]:
```diff
-The primary user facing change is step validation messages are different.
+The primary user facing change is different step validation messages.
```

**Comment by xtreme-sameer-vohra** [comment]
thanks @jomsie

**Pull request merged**